main.cpp

```diff
@@ -235,7 +235,7 @@ static void print_resources(const Compiler &compiler, const char *tag, const vec
 		if (mask & (1ull << DecorationInputAttachmentIndex))
 			fprintf(stderr, " (Attachment : %u)", compiler.get_decoration(res.id, DecorationInputAttachmentIndex));
 		if (is_sized_block)
-			fprintf(stderr, " (BlockSize : %u bytes)", block_size);
+			fprintf(stderr, " (BlockSize : %zu bytes)", block_size);
```

Comment:
%zu doesn't work on Windows unless recent msvcrt is used, I'd keep it as unsigned.